### ISSUE TITLE
publicPath option

### ISSUE THREAD

**Issue by staabm** [surma/rollup-plugin-off-main-thread#4]
it seems this plugin - while beeing announced as the successfor from loadz0r - does not support the `publicPath` as loadz0r did

https://github.com/surma/rollup-plugin-loadz0r/pull/11/files

was this removed intenionally or is it just missing right now?

**Comment by ansballard** [comment]
Looks like it is missing from this package. https://github.com/surma/rollup-plugin-loadz0r/blob/master/loader.ejs#L28 and https://github.com/surma/rollup-plugin-loadz0r/blob/master/loader.ejs#L68 are where it was used in loadz0r, but it looks like it was removed, intentionally or not. Should be as simple as adding the parameter [to the default options](https://github.com/surma/rollup-plugin-off-main-thread/blob/master/index.js#L19) (although I guess that's not necessary) and then copying in the above lines.

As a workaround, you _should_ be able to pass in your `publicPath` param as you did in loadz0r, then pass your own custom `loader.ejs` in the `loader` param that uses publicPath. You could probably even copy the one from loadz0r verbatim.

**Comment by kaitsh** [comment]
With the `import as URL` option I was able to solve this issue by changing the URL after the import:
```javascript
import workerURL from "omt:./worker.js";
let newURL = `./somePath/${workerURL}`

const worker = new Worker(newURL, { name: "main-worker" });
```

**Comment by jakearchibald** [comment]
Yeah I ran into this too. Let me know if you need more details @surma.
